source/applications/advanced/hand_eye_calibration/calibrate_eye_to_hand.py

```diff
@@ -7,14 +7,21 @@
 
 
 def _acquire_checkerboard_frame(camera):
+    print("Configuring settings...")
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(aperture=8.0, exposure_time=datetime.timedelta(microseconds=20000),),
+        ],
+        processing=zivid.Settings.Processing(
+            filters=zivid.Settings.Processing.Filters(
+                smoothing=zivid.Settings.Processing.Filters.Smoothing(
+                    gaussian=zivid.Settings.Processing.Filters.Smoothing.Gaussian(enabled=True)
+                ),
+            )
+        ),
+    )
```

Comment:
You can simply do this btw:
```
settings = zivid.Settings()
settings.acquisitions.append(zivid.Settings.Acquisition())
settings.acquisitions[0].aperture = 8.0
settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=20000)
settings.processing.filters.smoothing.gaussian.enabled = True
```
A little less parenthesis-hell, eh? :)

Reply:
done